guides/common/modules/proc_synchronizing-templates-with-a-local-directory.adoc

```diff
@@ -1,9 +1,7 @@
 [id="Synchronizing_Templates_with_a_Local_Directory_{context}"]
 = Synchronizing templates with a local directory
 
-Synchronizing templates with a local directory is useful if you have configured a version control repository in the local directory.
-That way, you can edit templates and track the history of edits in the directory.
-You can also synchronize changes to {ProjectServer} after editing the templates.
+If you store templates in a local directory that is tracked under a version control system, you can configure {Project} to synchronize the templates between your {ProjectServer} and the local directory.
```

Comment:
```suggestion
If you store templates in a local directory that is tracked under a version control system, you can synchronize the templates between your {ProjectServer} and the local directory.
```
This sounds like the project does this automatically, which, while it can be achieved, is not a purpose of this procedure.